pkg/epp/scheduling/types/types.go

```diff
@@ -32,6 +32,7 @@ type LLMRequest struct {
 	// Target models is a map of target model name to weight.
 	TargetModels map[string]int
 	Prompt       string
+	Headers      map[string]string
```

Comment:
can you add a comment with description like the other fields?

Reply:
Done. I also added one for MutatedHeaders in the SchedulingContext